dali/base/dadfs.cpp

```diff
@@ -8976,7 +8976,18 @@ class CInitGroups
     CConnectLock groupsconnlock;
     StringArray clusternames;
     unsigned defaultTimeout;
+    bool machinesLoaded;
 
+    GroupType getGroupType(const char *type)
+    {
+
+        if (0 == strcmp("ThorCluster", type))
+            return grp_thor;
+        else if (0 == strcmp("RoxieCluster", type))
+            return grp_roxie;
+        else
```

Comment:
I'm not sure, HThor 'clusters' aren't really clusters. The groups associated with them are a little odd, e.g. if you have a agent defined with multiple instances it creates a group per instance.
I'm not sure what you'd return as a group in that case, probably a group of all instances, but that wouldn't be much practical use to anyone I think.
I think I'll leave this anomaly for now, until/if anyone cares/needs it.